server/Cargo.toml

```diff
@@ -12,8 +12,10 @@ trustify-module-importer = { workspace = true }
 trustify-module-ingestor = { workspace = true }
 trustify-module-storage = { workspace = true }
 trustify-module-ui = { workspace = true }
+trustify-module-graphql = { workspace = true }
 
 actix-web = { workspace = true }
+actix-cors = { workspace = true }
```

Comment:
humor: You're going to draw the wrath and ire of @ctron by touching CORS. :)

Reply:
@bobmcwhirter, I removed those permissive rules ! So far so good.